src/components/Checkbox/Checkbox.jsx

```diff
@@ -0,0 +1,106 @@
+import React, {Component, PropTypes} from 'react';
+import {PURE} from 'dx-util/src/react/pure';
+import {themr} from 'react-css-themr';
+import classnames from 'classnames';
+import {randomId} from 'dx-util/src/string/string';
+
+import Icon from '../Icon/Icon.jsx';
+import defaultCheckboxIcon from './img/icon-checkbox-tick.svg';
+
+export const CHECKBOX = Symbol('Checkbox');
+
+@PURE
+@themr(CHECKBOX)
+export default class Checkbox extends Component {
+	static propTypes = {
+		id: PropTypes.string,
+		checkboxIconName: PropTypes.string,
+		isDisabled: PropTypes.bool,
+		isChecked: PropTypes.bool,
+		onChange: PropTypes.func,
+		theme: PropTypes.shape({
+			container: PropTypes.string,
+			container__disabled: PropTypes.string,
+			container__input: PropTypes.string,
```

Comment:
why do you need `container__` namespace each time? css-modules allow you to pick direct names like `input`